dts/bindings/sensor/meas,ms5607-spi.yaml

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2019, Thomas Schmid <tom@lfence.de>
+# SPDX-License-Identifier: Apache-2.0
+
+title: TE Connectivity digital pressure sensor MS5607
+
+description: >
+    This binding gives a base representation of the MS5607 digital pressure
+    sensor
```

Comment:
Know it's just to be consistent, but could put in a more meaningful description. I'm not even sure what "a base representation" was supposed to mean.

Could make it

```
description: |
    TE Connectivity MS6707 digital sensor. Datasheet at ...
```

Or whatever other information might be useful. Doesn't have to be super long.

The description shows up as a comment in `generated_dts_board_unfixed.h` above the generated macros.